src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -745,8 +740,7 @@ public void FileRegionsCache_PopulatesWithOneLine_IncreasingToTheRight()
             region = fileRegionsCache.PopulateTextRegionProperties(region, uri, true, content);
             Region multilineRegion = fileRegionsCache.ConstructMultilineContextSnippet(region, uri);
 
-            // CharLength + 128 to the right = 428 characters
-            multilineRegion.CharLength.Should().Be(300 + 128);
+            multilineRegion.CharLength.Should().Be(512);
```

Comment:
Curious, with the bug regarding `>` above, how does this test pass?

Reply:
I don't understand the bug that you see. big snippet is defined as 512.